substra/sdk/backends/local/backend.py

```diff
@@ -29,9 +31,17 @@
 
 class Local(base.BaseBackend):
     def __init__(self, backend, *args, **kwargs):
+        self._support_chainkeys = os.getenv("CHAINKEYS_ENABLED", False)
```

Comment:
I think we should add strtobool around the os.getenv so that `export CHAINKEYS_ENABLED=False` isn't interpreted as a true value here.

Reply:
That's a good catch, I did not know that function